x/leverage/types/token.go

```diff
@@ -115,7 +115,7 @@ func (t Token) Validate() error {
 		return sdkerrors.ErrInvalidRequest.Wrap("Token.MaxSupplyUtilization must be between 0 and 1")
 	}
 
-	if t.MinCollateralLiquidity.IsNegative() || t.MaxSupplyUtilization.GT(sdk.OneDec()) {
+	if t.MinCollateralLiquidity.IsNegative() || t.MinCollateralLiquidity.GT(sdk.OneDec()) {
```

Comment:
duplicate, I've solved this today morning in: https://github.com/umee-network/umee/pull/1409/files

I think we should merge #1409 to avoid migration when we add that feature. We don't need to use that value, now.

Reply:
up